rules-reviewed/jboss-eap/websphere/websphere-mq.windup.xml

```diff
@@ -0,0 +1,363 @@
+                </hint>
+            </perform>
+        </rule>
+
+
+        <rule id="websphere-mq-11000">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeJNDIConnectionFactory"/>
+            </when>
+            <perform>
+                <hint severity="Mandatory" effort="1">
+                    <message>To migrate to EAP 6, use the Java EE 6 JMS class `javax.jms.ConnectionFactory`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mq-12000">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeJNDIQueueConnectionFactory"/>
+            </when>
+            <perform>
+                <hint severity="Mandatory" effort="1">
+                    <message>To migrate to EAP 6, use the Java EE 6 JMS class `javax.jms.ConnectionFactory`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mq-16000">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeJMSQueue"/>
+            </when>
+            <perform>
+                <hint severity="Mandatory" effort="1">
+                    <message>To migrate to EAP 6, use the Java EE 6 JMS class `javax.jms.Queue`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mq-17000">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeJMSJNDIQueue"/>
+            </when>
+            <perform>
+                <hint severity="Mandatory" effort="1">
+                    <message>To migrate to EAP 6, use the Java EE 6 JMS class `javax.jms.Queue`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+
+        <rule id="websphere-mq-18000">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQe{type}"/>
+            </when>
+            <perform>
+                <hint severity="Mandatory" effort="1">
+                    <message>To migrate to EAP 6, use the Java EE 6 JMS class `javax.jms.{type}`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+
+
+        <!-- WebSphere JMS MQ -->
```

Comment:
The rules numbering will not always follow 1000's step. Does it really matter if we have that state now?